CHANGELOG.rst

```diff
@@ -1,3 +1,57 @@
+This project follows semantic versioning.
+
+Types of changes:
+
+- **Added**: New features.
+- **Changed**: Changes in existing functionality.
+- **Deprecated**: Soon-to-be removed features.
+- **Removed**: Removed features.
+- **Fixed**: Bug fixes.
+- **Infrastructure**: Changes in build or deployment infrastructure.
+- **Documentation**: Changes in documentation.
+
+[Unreleased]
+------------
+
+
+
+Release 0.11.0
+--------------
+
+Added
+#####
+
+- Add Python 3.8, 3.9, 3.10 support.
+- Improve error message on unmatched method signature expectation.
+- Add type annotations.
+
+Changed
+#######
+
+- Flexmock needs to be imported explicitly using ``from flexmock import flexmock``.
+
+Removed
+#######
+
+- Drop Python 2.7, 3.4, 3.5 support.
```

Comment:
I'm still not sure we should drop 3.4 and 3.5 already

Reply:
Why do you think so? Python 3.4 reached end of life already two years ago and Python 3.5 almost one year. Even pytest has dropped support for Python 3.4. If someone still needs Python 3.4 or 3.5, they can keep using an older flexmock version.

If we keep supporting Python 3.4 & 3.5 we would need to remove type hints.